hs_access_control/models.py

```diff
@@ -92,6 +92,19 @@ class PrivilegeCodes(object):
     # Names of privileges for printing
     NAMES = ('Unspecified', 'Owner', 'Change', 'View', 'None')
 
+    @classmethod
+    def from_string(self, privilege):
+        """ Converts a string representation to a PrivilegeCode """
+        if privilege.lower() == 'view':
+            return PrivilegeCodes.VIEW
+        if privilege.lower() == 'edit':
+            return PrivilegeCodes.CHANGE
+        if privilege.lower() == 'owner':
+            return PrivilegeCodes.OWNER
+        if privilege.lower() == 'none':
+            return PrivilegeCodes.NONE
+        return None
+
```

Comment:
isn't return None implicit aka not needed

Reply:
I have to return something and I figured None would be the least invasive.  I guess I could raise an exception that could be caught and return bad request.  Thinking of it now, that seems like a better solution.

Reply:
ok, I see what you mean now... I guess it is implicit, but it feels dirty to just omit.